bundles/org.openhab.binding.juicenet/src/main/resources/OH-INF/thing/juicenet-account.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="juicenet"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="account">
+		<label>JuiceNet Account</label>
+		<description>JuiceNet account</description>
```

Comment:
Could the description be extended for being a bit more descriptive than the label?